operators/test/e2e/stack/builder.go

```diff
@@ -48,12 +50,12 @@ func NewStackBuilder(name string) Builder {
 		Name:      name,
 		Namespace: params.Namespace,
 	}
-
 	return Builder{
 		Elasticsearch: estype.Elasticsearch{
 			ObjectMeta: meta,
 			Spec: estype.ElasticsearchSpec{
-				Version: params.ElasticStackVersion,
+				SetVMMaxMapCount: helpers.BoolPtr(false),
```

Comment:
does this mean we are never testing the setvmmaxmapcount init container in E2E tests?

Reply:
`SetVMMaxMapCount` is true by default since https://github.com/elastic/cloud-on-k8s/commit/7a55d44a15bbedebd36164c63f7c5c6c6c86ec52

Reply:
Aren't we explicitly making it `false` for all integration tests here (since they are all using the default stack Builder)?

Reply:
Yes because with the restricted PSP containers are not allowed to run as root anymore in e2e tests.